usr/src/cmd/zoneadmd/log.c

```diff
@@ -444,9 +444,10 @@ logstream_init(zlog_t *zlogp)
 		lfp->lf_fd = -1;
 		if (custr_alloc_buf(&lfp->lf_cus, lfp->lf_buf,
 		    sizeof (lfp->lf_buf)) != 0) {
-			zerror(zlogp, B_FALSE, "failed to allocate custr_t "
-			    "for log file");
-			goto fail;
+			(void) fprintf(stderr, "failed to allocate custr_t for "
+			    "log file\n");
+			(void) fflush(stderr);
```

Comment:
stderr is unbuffered

Reply:
Yeah -- I've had past reviews where the reviewer felt that it shouldn't be assumed and would call out the lack of fflush as an error.

Reply:
(I think if you've done something like that, you shouldn't be surprised if there are problems, but also not something I thought was important enough to hold things up).. but since that's not the situation here, I've removed the calls since they are unnecessary...

Reply:
My feeling is if it's in the manpage, and we're in a situation where the case it's *not* true is of minimal impact, it's fine to have code make reasonable assumptions about its operating environment.

Reply:
My feeling is if it's in the manpage, and we're in a situation where the case it's *not* true is of minimal impact, it's fine to have code make reasonable assumptions about its operating environment.